napari_aicssegmentation/widgets/workflow_thumbnails.py

```diff
@@ -109,6 +110,8 @@ def _add_buttons(self, workflows: List[WorkflowDefinition]):
             button.setFixedSize(PAGE_CONTENT_WIDTH, 200)
 
             button.setEnabled(False)
+            button.setObjectName(workflow.name)
+            button.clicked.connect(self.view.combo_workflow_activated)
```

Comment:
Passing in the view like this is somewhat of an anti-pattern. I think what we want to do is expose a callable / event on this Widget for the view to connect to. Might take a while to figure out how to do this properly in a QT friendly way, so I think this is fine for now, but can we please add a TODO comment to come back and refactor later?